packages/toggle-switch/toggle-slider.scss

```diff
@@ -23,117 +24,147 @@ $jkl-slider-rounded: rem(100px);
+        padding: $component-spacing--small 0;
+        position: relative;
         border-radius: $jkl-slider-rounded;
-        box-shadow: 0 0 rem(4px) rgba($svart, 0.15);
-    }
+        overflow: hidden;
+        display: flex;
+        justify-content: space-between;
+        max-width: $jkl-slider-width;
+        width: 100%;
+        user-select: none;
+
+        &__pill {
+            width: 50%;
+            height: calc(100% - #{$jkl-slider-offset} - #{$jkl-slider-offset});
+            background-color: $hvit;
+            background-color: var(--jkl-slider-pill);
+            display: block;
+            position: absolute;
+            top: $jkl-slider-offset;
+            transform: translateX($jkl-slider-offset__pill--right);
+            transition-property: transform;
+            @include motion(standard, expressive);
+            border-radius: $jkl-slider-rounded;
+            box-shadow: 0 0 rem(4px) rgba($svart, 0.15);
+        }
 
-    &--left,
-    &--right {
-        z-index: 1;
-        display: block;
-        width: 50%;
-        text-align: center;
-        color: $gra-60;
-        color: var(--jkl-slider-pill-color);
-        transition-property: color;
-        @include motion(standard, expressive);
-    }
+        &--left,
+        &--right {
+            z-index: 1;
+            display: block;
+            width: 50%;
+            text-align: center;
+            color: $gra-60;
+            color: var(--jkl-slider-pill-color);
+            transition-property: color;
+            @include motion(standard, expressive);
+        }
 
-    &--left {
-        margin-left: calc(#{$jkl-slider-offset} * 2);
-    }
-    &--right {
-        margin-right: calc(#{$jkl-slider-offset} * 2);
-    }
+        &--left {
+            margin-left: calc(#{$jkl-slider-offset} * 2);
+        }
+        &--right {
+            margin-right: calc(#{$jkl-slider-offset} * 2);
+        }
 
-    &--checked {
-        & > .jkl-toggle-slider--right {
-            color: $svart;
-            color: var(--jkl-slider-pill-color--active);
+        &--left,
+        &--right,
+        &__label {
+            @include jkl-text-style("compact/small");
+            font-weight: $bold-weight;
         }
-    }
 
-    &--not-checked {
-        & > .jkl-toggle-slider__pill {
-            transform: translateX($jkl-slider-offset__pill--left);
+        @include from-large-device {
+            &--left,
+            &--right,
+            &__label {
+                @include jkl-text-style("desktop/small");
+                font-weight: $bold-weight;
+            }
         }
-        & > .jkl-toggle-slider--left {
-            color: $svart;
-            color: var(--jkl-slider-pill-color--active);
+
+        &--checked {
+            & > .jkl-toggle-slider--right {
+                color: $svart;
+                color: var(--jkl-slider-pill-color--active);
+            }
+
+            @media (hover: hover) {
```

Comment:
Dette for å unngå hover-effekt på mobil etter å ha klikket på et element.

Reply:
:not([data-touchnavigation]) er kanskje mer robust for dette.